apps/roam/src/components/DiscourseNodeMenu.tsx

```diff
@@ -260,25 +262,32 @@ export const NodeMenuTriggerComponent = (
   }, [comboKey]);
 
   return (
-    <InputGroup
-      inputRef={inputRef}
-      placeholder={isActive ? "Press keys ..." : "Click to set trigger"}
-      value={shortcut}
-      onKeyDown={handleKeyDown}
-      onFocus={() => setIsActive(true)}
-      onBlur={() => setIsActive(false)}
-      rightElement={
-        <Button
-          hidden={!comboKey.key}
-          icon={"remove"}
-          onClick={() => {
-            setComboKey({ modifiers: 0, key: "" });
-            extensionAPI.settings.set("personal-node-menu-trigger", "");
-          }}
-          minimal
-        />
-      }
-    />
+    <div>
+      <p className="mb-2 text-neutral-dark/80">
```

Comment:
Let's use the tooltip pattern that the rest of the settings currently use for additional information.

![image](https://github.com/user-attachments/assets/8b6b71c5-a784-4254-9d29-1333cf6c35cc)